### PR TITLE
Fix -fail messages appearing with Circle Throw/Dragon Tail

### PR DESCRIPTION
**Pull request by ascriptmaster**

### CODE REVIEW

File: data/scripts.js [hunk 1]
```diff
@@ -549,7 +549,7 @@ exports.BattleScripts = {
 			hitResult = this.runEvent('DragOut', target, pokemon, move);
 			if (hitResult) {
 				target.forceSwitchFlag = true;
-			} else if (hitResult === false) {
+			} else if (hitResult === false && move.status === 'Status') {
```
Comment (Rhythms88) [inline data/scripts.js]: I'm confused. Shouldn't this be `move.category`? Someone please correct me if I'm incorrect.

Comment (Zarel) [inline data/scripts.js]: @Rhythms88 is correct.

**Comment by ascriptmaster** [comment]
kk i'm dumb

**Comment by Rhythms88** [comment]
n ur not

**Pull request merged**